README.md

```diff
@@ -29,3 +29,44 @@ To test if it is installed correctly,
 ```julia
 pkg> test ArchGDAL
 ```
+
+## Dependencies
+To manage the dependencies of this package, we work with [environments](https://pkgdocs.julialang.org/v1.6/environments/):
+
+1. Navigate to the directory corresponding to the package:
+
+```julia
+shell> cd /Users/yeesian/.julia/dev/ArchGDAL
+/Users/yeesian/.julia/dev/ArchGDAL
+```
+
+2. Activate the environment corresponding to `Project.toml`):
+
+```julia
+(@v1.6) pkg> activate
```

Comment:
Are you missing a . here?
Otherwise this recommendation would install all dependencies into the global environment and this is not what we want here.

Reply:
Haha yes, thank you! :)